File: 이것이 취업을 위한 코딩 테스트다/정렬/정렬_라이브러리/정렬_라이브러리.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main() {
  int arr[] = {7, 5, 9, 0, 3, 1, 6, 2, 4, 8};
  int size = sizeof(arr) / sizeof(arr[0]);
  sort(arr, arr + size);

  for (int i = 0; i < size; i++) {
    cout << arr[i] << ' ';
  }

  return 0;
}

File: 이것이 취업을 위한 코딩 테스트다/정렬/정렬_라이브러리/정렬_라이브러리.java
import java.util.*;

public class 정렬_라이브러리 {
  public static void main(String[] args) {
    int[] arr = {7, 5, 9, 0, 3, 1, 6, 2, 4, 8};
    Arrays.sort(arr);

    for (int num : arr) {
      System.out.print(num + " ");
    }
  }
}
